examples/structured_2d_dgsem/elixir_shallowwater_three_mound_dam_break.jl

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
This three mound problem needs a rectangular mesh. So it should use the `UnstructuredMesh` instead. This would also mean that we only add one new elixir rather than two. Once the unstructured version is working we can remove the `TreeMesh` and `StructuredMesh` versions.